tests/config/nf-test.config

```diff
@@ -6,7 +6,7 @@ params {
 
 process {
     cpus = 4
-    memory = 5.GB
+    memory = 8.GB
```

Comment:
Same thing as above, revert it to 5.GB and try modifying it in your config file.